draft-barnes-hpke-hpke.md

```diff
@@ -412,6 +440,12 @@ this KEM algorithm; if used in the remainder of HPKE, it MUST start with
 "HPKE" and identify the entire ciphersuite in use. See sections {{dhkem}}
 and {{encryption-context}} for details.
 
+Certain functions have a different structure depending on whether a one-stage or
+two-stage KDF is being used.  Implementations of such functions will have the
```

Comment:
Are we defining an API?  Perhaps say "for clarity in this document, the definition and use of such functions will have suffixes ..."

Reply:
Reworded.